py/README.md

```diff
@@ -139,26 +241,5 @@ lncli-ws openchannel --node_key=<address_of_A> --local_amt=1000000
 ```
 
 This channel should get listed in `lncli-ws pendingchannels`.
-Mine 6 blocks to confirm this channel: `lncli-ws generate 6`.
+Mine 6 blocks to confirm this channel: `./btcctl generate 6`.
 We should be able to see the channel in: `lncli-ws listchannels` now.
-
-### Sending a payment
```

Comment:
maybe not remove this?

Reply:
I didn't remove it, not sure why it says it is removed. I did move the opening a channel part down to the appendix since it wasn't the exact process anymore, but I thought it was still helpful to have in the readme